app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id', dependent: :destroy
+  has_many :comments, foreign_key: 'author_id', dependent: :destroy
+  has_many :likes, foreign_key: 'author_id', dependent: :destroy
+
+  def recent_three
```

Comment:
- [ ] [OPTIONAL] I would like to suggest that you give your methods names that can be easily understood by other developers going through your codebase. It will be nice to give this method
``` ruby
def recent_posts
```